docker/puppetdb/docker-entrypoint.d/20-wait-for-hosts.sh

```diff
@@ -33,6 +33,14 @@ wait_for_host_port() {
   fi
 }
 
+wait_for_host_or_port() {
```

Comment:
We talked yesterday about improving the DNS query bit... I think we might end up installing `dnsutils` package so that we have `dig` available. Then we can get better error messages when it's a config / DNS problem.

Will try to get some variation of this PR in ASAP for you though - thanks for the submission!

Reply:
That sounds like a great idea. Thanks for looking into this!

Reply:
I have #2935 up which hopefully does the trick for you!